File: week02/coke/coke.py
amountDue = 50

while(amountDue > 0):
    print(f"Amount Due: {amountDue}")
    coin = int(input("Insert Coin: "))
    if((coin == 25) or (coin == 10) or (coin == 5)):
        amountDue = amountDue - coin

changeOwed = abs(amountDue)
print(f"Change Owed: {changeOwed}")

File: week02/coke/coke.c
#include <stdio.h>
#include <stdlib.h>

int main(void)
{
    int amountDue = 50;
    int coin;

    while(amountDue > 0)
    {
        printf("Amount Due: %d\n", amountDue);
        printf("Insert Coin: ");
        scanf("%i", &coin);
        if((coin == 25) || (coin == 10) || (coin == 5))
        {
            amountDue = amountDue - coin;
        }
    }

    int changeOwed = abs(amountDue);
    printf("Change owed: %i\n", changeOwed);

}
